src/interfaces/ubi/ubiCommunityContract.ts

```diff
@@ -33,6 +33,8 @@ export interface UbiCommunityContract {
     maxClaim: string;
     baseInterval: number;
     incrementInterval: number;
+    blocked?: boolean;
+    decreaseStep?: string;
```

Comment:
I also think this should be blocked: false and decreaseStep: 0 by default, simplifying further implementations no need to check possibly undefined or forcing defined)